### PR TITLE
Some code clean up.

### PR DESCRIPTION
**Pull request by dbtsai**

### CODE REVIEW

**Comment by rxin** [comment]
LGTM.

**Comment by rxin** [comment]
Jenkins, retest this please.

File: mllib/src/main/scala/org/apache/spark/mllib/util/LinearDataGenerator.scala [hunk 1]
```diff
@@ -105,8 +105,7 @@ object LinearDataGenerator {
     val x = Array.fill[Array[Double]](nPoints)(
       Array.fill[Double](weights.length)(rnd.nextDouble()))
 
-    x.foreach {
-      case v =>
+    x.foreach { v =>
         var i = 0
```
Comment (cloud-fan) [inline mllib/src/main/scala/org/apache/spark/mllib/util/LinearDataGenerator.scala]: Should change the indent too.

Comment (dbtsai) [inline mllib/src/main/scala/org/apache/spark/mllib/util/LinearDataGenerator.scala]: good catch!

**Comment by rxin** [comment]
Thanks. I've merged this in master.

**Pull request closed without merging**